src/systems/geometry.js

```diff
@@ -132,6 +132,7 @@ function toBufferGeometry (geometry, doBuffer) {
   if (!doBuffer) { return geometry; }
 
   bufferGeometry = new THREE.BufferGeometry().fromGeometry(geometry);
+  bufferGeometry.oldMetadata = {type: geometry.type, parameters: geometry.parameters || {}};
```

Comment:
Was thinking we name it `metadata` since the data is still current, but just destroyed.